### ISSUE TITLE
Fullscreen

### ISSUE THREAD

**Issue by zh-wowtv** [aslanyanhaik/youtube-iOS#8]
Are you going to support video play in fullscreen when rotate to landscape?

**Comment by aslanyanhaik** [comment]
@zh-wowtv yes, in upcoming weeks will update the source code and add portrait feature along with seeker bar

**Comment by dmyma** [comment]
Seems like the MainVIewCollectionCell does not really follow the constraints in code, more the cell frame in the storyboard
